test/sanity/null.wtest

```diff
@@ -71,6 +71,7 @@ describe "Null Test Case"{
 	
 			assert.notThat(8 == null)
 			assert.notThat("pepe" == null)
+			// TODO: ?
```

Comment:
This is uncommented on my branch, so this test is passing ok

Reply:
Removing comment.